src/doc/rustdoc/src/what-is-rustdoc.md

```diff
@@ -93,6 +93,22 @@ passes `-L`, a flag that helps rustdoc find the dependencies
 your code relies on. If our project used dependencies, we'd get
 documentation for them as well!
 
+## Crate level documentation
+From the above examples, you can see that using the `///` syntax
+documents the next item. But that can't be used when you are
+documenting the entire crate, because there is nothing before the crate.
+It is where everything is placed under. So, in order to document an entire
+crate, you use this `//!` syntax. For example
+
+``` rust
+//! This is my first rust crate
+```
+
+Actually the `//!` syntax is used to document the item it is present
+in. So by using this at the crate root, we want to document the item it
+is present in, which is the crate itself. You can use it inside functions,
+structs as well.
```

Comment:
```suggestion
The `//!` syntax is used to document the item it is present in,
so when used at the crate root, it documents the crate itself.
You can use it anywhere that is inside a larger item, such as
`struct` implementations, function bodies, and modules.
```

Reply:
Oh actually I missed that https://doc.rust-lang.org/book/ch14-02-publishing-to-crates-io.html?highlight=Document#commenting-contained-items already exists - we should link there rather than repeating the info.

Reply:
You marked this as resolved but didn't apply the second suggestion.

Reply:
I am just doing it

Reply:
Gotcha. Please wait to mark this as resolved until you've made the change in the future.